### DISCUSSION TITLE
Colab Python Pandas notebook with some initial data for challenge #3

### DISCUSSION THREAD

**Discussion by kodiakpony** [datakind/datakit-housing-fall-2024#18] [Show and tell]
I mentioned this in Slack but figured I should do it here, too. Here's a [Google Colab notebook](https://colab.research.google.com/drive/1G_lhVn900ZugQ0xoan6127PGQ942UEvz?usp=sharing) that I'm planning to use to explore the dataset using python pandas, which may be a useful starting point for others of you taking this approach.  So far it just:

-  Grabs the ZIP file from DataKind's datakit-housing-fall-2024 GitHub
-  Reads the Florida csv
-  Creates a Pandas dataframe with just the subset of columns related to HUD, CDFI and LIHTC opportunity zones, along with economic distress, geoid and county
-  Makes the column types consistent: YES->1, NO->0
-  Performs some cross-tabulation to see how they compare

Initial observations:   
A lot of the "investment_areas" are in geoids that are not economically distressed.  (377 entries have economic_distress_pop_agg=0 and investment_areas=1 vs 512 with both=1).  This is less true for qct and opzone.
There is quite a bit of overlap - entries that are in more than one of investment_areas, qct, and opzone 

Here are a couple of screen shots to see what I'm up to:
![screenshot_df](https://github.com/user-attachments/assets/e2f8828b-a121-4e47-bbc2-0a49ea2635bf)
![screenshot_crosstabs](https://github.com/user-attachments/assets/3a06207b-a991-4eb5-961c-a19486fc9ff8)

**Comment by ssenhouse** [comment]
Hey @kodiakpony, thanks for sharing your notebook. I took a look at it and very good stuff. From your Venn diagram, is there a way to identify the zip code or town where the economic distress and investment areas overlap? You've identified 512 of those areas, and I thought that might be a great way to start to illustrate the investment that has been happening in those areas.

**Reply by kodiakpony** [reply]
Good idea. And thanks for taking a look!   The dataset I'm using has geoid and county_fips_code fields, so I can easily find out which of those are in the overlap, but I'm not sure how to translate those ids into something user friendly (like county names, towns, and zip codes). I'm sure the data must exist somewhere, perhaps in one of the other datasets mentioned in the challenge.  If anyone has already figured that out, please let me know!  I'm also interested in visualizing the data on a map, but would love some pointers since I haven't done much geospatial analysis before (but would love to learn)
